circus/util.py

```diff
@@ -144,7 +144,12 @@ def bytes2human(n):
 
     for s in reversed(_SYMBOLS):
         if n >= prefix[s]:
-            value = int(float(n) / prefix[s])
+            # truncate to 2 decimal places without rounding
+            # https://stackoverflow.com/questions/783897/truncating-floats-in-python
```

Comment:
What would be wrong with rounding though? (`round(float(n) / prefix[s], 2)`)

Reply:
`round(..., 2)` for `1024 * 1024 * 2047` will give `2.00G`, but it is in fact `1.99G`.

Reply:
I don't see what's wrong with `2.00G` in this case. In the original issue we were displaying `1G`, which was plain wrong. But here `2.00G` is the right rounding IMO.

Reply:
Ok, no problem. I was just trying to reproduce the example from the issue.

I changed to round, fixed tests and also made readings fixed width, because it was printing `2.0G` after rounding and not `2.00G` as with all other values.